extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -126,6 +134,15 @@ export class IframeTransport {
           /** @type {!HTMLIFrameElement} */
           (frame)),
     });
```

Comment:
The reason I am creating this PR now, when there is a known bug, is because I want to be able to refer to the bug that currently is in the following code, over in the other PR, #10714.
1. The root issue here is that we can't call listenFor() after the frame has already been added to the DOM. (That is done on line 84.) See https://github.com/ampproject/amphtml/blob/ea4b3f60b0c6293d4fb31f100886b666cb1f398b/src/iframe-helper.js#L252 for why this is not allowed.
2. So, we have to call listenFor() *before* adding the frame to the DOM. This means that if 2 or more creatives use the same xframe, only the first can be the listener.
3. Therefore, if creative #1 receives a response, it might be for a different creative.
4. This was addressed in a previous, unmerged, PR (see https://github.com/google/amphtml/blob/eabf92dd9b0f431e676a985cc1c83506611aa5ef/extensions/amp-analytics/0.1/transport.js#L244) by having a static structure attached to IframeTransport, which allowed the first creative to dispatch the response to the actual intended destination creative.
My intention is to more-or-less copy the code described in item 4 to the current PR.